index.html

```diff
@@ -1,5 +1,8 @@
 <!DOCTYPE html>
-<html lang="en">
+<html
+  lang="en"
+  class="h-full bg-gradient-to-b from-interYellow to-interOrange bg-no-repeat bg-fixed"
```

Comment:
why class the `<html>` instead of `<body>`?

Reply:
For the background color, I want it to affect the entire page even if there's no content. This way we don't have to artificially stretch the body out to fill the whole screen just to apply the background.